src-java/apis/src/main/java/test.java

```diff
@@ -50,5 +56,27 @@ public static void main(String[] args){
             response.body("Hello RPIEats");
             return response;
         });
+
+        get("/restaurants", (request, response) -> {
+            //String name = request.params(":name");
```

Comment:
Take these commented lines out (61-62), and I'll approve it and merge the PR.

Reply:
Done!